import_mu.py

```diff
@@ -29,7 +29,7 @@
 
 from mu import MuEnum, Mu, MuColliderMesh, MuColliderSphere, MuColliderCapsule
 from mu import MuColliderBox, MuColliderWheel
-from shader import make_shader
+from shader import make_material
```

Comment:
Who is "we"? My point is that we (probably) don't want to diverge from the original repo. So either we contact the main maintener to make sure he would be ok to merge this or we add a flag.